apcd-cms/src/apps/submitter_renewals_listing/templates/list_submitter_registrations.html

```diff
@@ -0,0 +1,126 @@
+                  <td>
+                    {% include "view_registration_modal.html" %}
+                    <select id='actionsSubmitterDropdown_{{r.reg_id}}' class='status-filter' onchange="openAction('{{r.reg_id}}')">
+                      <option value="">Actions</option>
+                      <option value="viewRegistration">View Record</option>
+                      <option value="renewRegistration">Renew Registration (coming soon)</option>
+                    </select>
+                  </td>
+              </tr>
+          {% endfor %}
+      </tbody>
+  </table>
+  {% include 'paginator.html' %}
+</div>
+<script>
+  function filterTableByStatus() {
+    var dropdown, statusFilter, xhr, url_params, url;
+    dropdown = document.getElementById("statusFilter");
+    statusFilter =  dropdown.value;
+    url_params = `?status=${statusFilter}`;
+    {% if selected_org %}
+      url_params += `&org={{selected_org}}`;
+    {% endif %}
+    url = `/register/list-registration-requests/${url_params}`;
+    xhr = new XMLHttpRequest();
+    xhr.open('GET', url);
+    xhr.send();
+    window.location.href = url;
+    window.location.load();
+  }
+  function filterTableByOrganization() {
+    var input, orgFilter, xhr, url_params, url;
+    input = document.getElementById("organizationFilter");
+    orgFilter = input.value.replace("&", encodeURIComponent('&'));
+    url_params = `?org=${orgFilter}`;
+    {% if selected_status %}
+      url_params = `?status={{selected_status}}&org=${orgFilter}`;
+    {% endif %}
+    url = `/register/list-registration-requests/${url_params}`;
+    xhr = new XMLHttpRequest();
+    xhr.open('GET', url);
+    xhr.send();
+    window.location.href = url;
+    window.location.load();
+  }
+  function clearSelections() {
+        var xhr;
+        xhr = new XMLHttpRequest();
+        xhr.open('GET', '/register/list-registration-requests/')
+        xhr.send()
+        window.location.href = '/register/list-registration-requests/';
+        window.location.load();
+    }
+
+    function openAction(reg_id) {
+      var actionsDropdown, selectedOption, modal_id;
+      actionsDropdown = document.getElementById(`actionsSubmitterDropdown_${reg_id}`);
+      /* grabs dropdown option number selected by user via selectedIndex, 
+        then grabs actual value associated with that option via 
+        options[index].value */
+      selectedOption = actionsDropdown.options[actionsDropdown.selectedIndex].value;
+      modal_id = `${selectedOption}Modal_${reg_id}`;
+      $(`#${modal_id}`).modal({backdrop: "static"}); /* modal appears manually */
+      actionsDropdown.selectedIndex = 0; /* resets dropdown to display 'Select Action' again */
+      if (selectedOption == "renewRegistration") {
+        var xhr;
+        xhr = new XMLHttpRequest();
+        xhr.open('GET', `/register/request-to-submit/${reg_id}`)
+        xhr.send()
+        window.location.href = `/register/request-to-submit/${reg_id}`;
```

Comment:
```suggestion
        xhr.open('GET', `/register/request-to-submit/?reg_id=${reg_id}`)
        xhr.send()
        window.location.href = `/register/request-to-submit/?reg_id=${reg_id}`;
```
I think the registration form view is looking for the var name `reg_id`, so this should fix that to where the link is no longer broken.